app/logger.js

```diff
@@ -0,0 +1,45 @@
+import pino from 'hapi-pino'
+
+const transport = { target: 'pino-pretty' }
```

Comment:
@aaroncarroll82 has been adding a couple of options to this transport on other repos https://github.com/DEFRA/ffc-ahwr-dashboard/blob/main/app/plugins/logger.js#L3.

Reply:
Nice, will use those settings